tomls/omnibus-arbitrum-sepolia/spot/USDe.toml

```diff
@@ -0,0 +1,70 @@
+[var.USDe_spot_settings]
+synth_USDe_max_collateral_amount = "<%= parseEther(String(1_000)) %>"
+synth_USDe_skew_scale = "<%= parseEther(String(350_000)) %>"
```

Comment:
@kaleb-keny we dont use skew configs here (maybe we should?!). And please update these values (copied from eth spot)

Reply:
updated, thank you